Framework/PythonInterface/plugins/algorithms/GSASIIRefineFitPeaks.py

```diff
@@ -113,7 +115,9 @@ def PyInit(self):
 
         self.declareProperty(name=self.PROP_PAWLEY_DMIN, defaultValue=1.0, direction=Direction.Input,
                              doc="For Pawley refiment: as defined in GSAS-II, the minimum d-spacing to be used in a "
-                                 "Pawley refinement. Please refer to the GSAS-II documentation for full details.")
+                                 "Pawley refinement. Please refer to the GSAS-II documentation for full details. Note, "
+                                 "if this corresponds to a TOF value less than {} or the lowest TOF value in the data, "
+                                 "the greatest of the 3 values is used".format(self.PROP_XMIN))
```

Comment:
If adding `PawleyDMin=1000` to the test script, one gets a bit messy error `RuntimeError: index -1 is out of bounds for axis 0 with size 0`. Would it be possible to add some validation to this parameter to get more meaningful error messages?

Reply:
Another question: should the doc read `if this corresponds to a TOF value less than self.PROP_XMIN or the lowest TOF value in the data the greatest of the 3 values is used as self.PROP_XMIN`?